tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -424,6 +451,9 @@ def test_common_workload(self):
                 "auto.offset.reset": "earliest",
                 "enable.auto.commit": "false",
             },
+            "topic_config": {
```

Comment:
I notice there is no batch.size in the "producer_config" above, does it mean only linger is applied or there is a default batch size somewhere?